src/components/cards/standard/StandardCard.tsx

```diff
@@ -2,13 +2,14 @@ import { useCardAnalyticsCallback } from '../../../hooks/useCardAnalyticsCallbac
 import { CardProps } from '../../../models/cardComponent';
 import { useCardFeedbackCallback } from '../../../hooks/useCardFeedbackCallback';
 import { StandardCardDisplay, StandardCardCssClasses } from './StandardCardDisplay';
+import { DefaultRawDataType } from '../../../models';
 
 /**
  * Props for a StandardCard.
  *
  * @public
  */
-export interface StandardCardProps extends CardProps {
+export interface StandardCardProps<T = DefaultRawDataType> extends CardProps<T> {
```

Comment:
does this need a default if we're specifying type `any` in `StandardCard` below? is it in case people want to extend this interface? 

should the same be done for `StandardSectionProps` to be consistent?

Reply:
yea, in case people extend it, but I don't think anyone really would. No harm to having a default though